docs/iframe.html

```diff
@@ -58,4 +58,4 @@
   } catch (e) {
     // eslint-disable-next-line no-console
     console.warn('unable to connect to parent frame for connecting dev tools');
-  }</script></head><body><div class="sb-nopreview sb-wrapper"><div class="sb-nopreview_main"><h1 class="sb-nopreview_heading sb-heading">No Preview</h1><p>Sorry, but you either have no stories or none are selected somehow.</p><ul><li>Please check the Storybook config.</li><li>Try reloading the page.</li></ul><p>If the problem persists, check the browser console, or the terminal you've run Storybook from.</p></div></div><div class="sb-errordisplay sb-wrapper"><div id="error-message" class="sb-heading"></div><pre class="sb-errordisplay_code"><code id="error-stack"></code></pre></div><div id="root"></div><script src="runtime~main.88b81c10ddab76505503.bundle.js"></script><script src="vendors~main.88b81c10ddab76505503.bundle.js"></script><script src="main.88b81c10ddab76505503.bundle.js"></script></body></html>
\ No newline at end of file
+  }</script></head><body><div class="sb-nopreview sb-wrapper"><div class="sb-nopreview_main"><h1 class="sb-nopreview_heading sb-heading">No Preview</h1><p>Sorry, but you either have no stories or none are selected somehow.</p><ul><li>Please check the Storybook config.</li><li>Try reloading the page.</li></ul><p>If the problem persists, check the browser console, or the terminal you've run Storybook from.</p></div></div><div class="sb-errordisplay sb-wrapper"><div id="error-message" class="sb-heading"></div><pre class="sb-errordisplay_code"><code id="error-stack"></code></pre></div><div id="root"></div><script src="runtime~main.927f493e28377c6b7131.bundle.js"></script><script src="vendors~main.927f493e28377c6b7131.bundle.js"></script><script src="main.927f493e28377c6b7131.bundle.js"></script></body></html>
```

Comment:
What do these iframe files do. I don't remember seeing them in one of these P/Rs before.

Reply:
This is code generated by storybook, which is used for documentation.